README.md

```diff
@@ -12,6 +12,14 @@ All available domains can be found in [src folder](https://github.com/ihmpavel/f
 
 `IsEmailOptions` are available options for email validation from [validator](https://www.npmjs.com/package/validator) library. You can check [test folder](https://github.com/ihmpavel/free-email-domains-list/tree/master/src) for usage.
 
+If you don't want to install `validator` as a peer dependency, you can also just import the list of domains like so:
+
+```ts
+import { DOMAINS } from 'free-email-domains-list/build/src/constants';

```

Comment:
From my testing, `isFreeEmail`, `isCompanyEmail` and `DOMAINS` should be importable without going directly to `build/src/*`.

Can you please re-test that with my suggestion if that works also for you?

```suggestion
import { DOMAINS } from 'free-email-domains-list'
```

Reply:
When you `import { DOMAINS } from 'free-email-domains-list'`, you will get a runtime error if you don't have `validator` installed.